resources/templates/templates.json

```diff
@@ -71,44 +71,9 @@
             "type": "AzureFunctions"
         },
         {

```

Comment:
I think there are other text you need to update.  like  'MXCHIP IoT DevKit'

Reply:
Fixed.